react/__mocks__/vtex.styleguide.js

```diff
@@ -0,0 +1,9 @@
+import React, { Fragment } from 'react'
+
+export function Button(props) {
+  return <div> {props.children} </div>
```

Comment:
```suggestion
  return <div class="button"> {props.children} </div>
```
Or
```suggestion
  return <button type="button"> {props.children} </button>
```
Or mocks should be a little more inteligent to create better snapshots.